SPECS/iptables/ip4save

```diff
@@ -1,13 +1,12 @@
 # init
 *filter
-:INPUT DROP [0:0]
-:FORWARD DROP [0:0]
-:OUTPUT DROP [0:0]
-# Allow local-only connections
+:INPUT ACCEPT [0:0]
```

Comment:
This leaves the firewall open by default from the outside.  Any reasons for this, considering that holes are poked through for things like SSH below?